src/components/DayOverview/DayOverview.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import supabase, { Schema } from "@/supabase";
+import DayOverviewButton from "@/components/DayOverview/DayOverviewButton";
+
+interface Props {
+    text: string;
+    date: Date;
+    location: string;
+}
+
+export type ParcelOfSpecificDateAndLocation = {
+    collection_datetime: Schema["parcels"]["collection_datetime"];
```

Comment:
Is this not just a date? (or a string idk which lol)

Reply:
I would say to change this, I think it is implied that it is the same through the name of the variable.

Reply:
Also the fact I don't immediately know what the type is isn't great

Reply:
But if this is the case then the `Pick` stuff also doesn't immediately tell us the type
But hard coding all the types doesn't seem like a good idea
The idea is we should refer to the Schema to check the type? I ain't sure - let's see how others think

Reply:
I think @ChloeYiuSoftwire is correct here, I would keep it as it is